Update Marketo Static Lists and DV360 deprecation notices. (#6685)
segmentio/segment-docs: src/connections/destinations/catalog/personas-display-video-360/index.md

---

> warning "Deprecation Notice"

> > Due to Google retiring certain APIs on March 6, 2024, Segment has deprecated this destination. Segment has created an instance of the [Display and Video (Actions)](/docs/connections/destinations/catalog/actions-display-video-360/) destination for each properly configured version of the Personas  Google  Display and Video 360 classic destination in your workspace. 

Reviewer: ```suggestion
> Due to Google retiring certain APIs on March 6, 2024, Segment deprecated this destination. Segment created an instance of the [Display and Video (Actions)](/docs/connections/destinations/catalog/actions-display-video-360/) destination for each properly configured version of the Personas Google Display and Video 360 classic destination in your workspace. 
```

> 

> > Settings and configurations have been migrated automatically, but you must take additional action to ensure the destination is properly functioning. For more information, see [Migrate from Personas Google Display & Video 360 Destination ](/docs/connections/destinations/catalog/actions-display-video-360/#migrate-from-personas-google-display-&-video-360-destination)

Reviewer: ```suggestion
> Segment automatically migrated destination settings and configurations, but you must take additional action to ensure your migrated destination functions as intended. For more information, see [Migrate from Personas Google Display & Video 360 Destination ](/docs/connections/destinations/catalog/actions-display-video-360/#migrate-from-personas-google-display-&-video-360-destination).
```

> 

> > Segment has disabled all existing Personas Display and Video 360 destinations. You can still access your existing configuration, but please refrain from enabling the destination, as it has been deprecated. You will no longer be able to create new instances of Personas Display and Video 360. Please check out [Display and Video (Actions)](/docs/connections/destinations/catalog/actions-display-video-360/) to set up a new instance of Google Display and Video 360. 

Reviewer: ```suggestion
> Segment disabled all existing Personas Display and Video 360 destinations. You can still access your existing configuration, but do not enable the destination, as it has been deprecated. You will no longer be able to create new instances of Personas Display and Video 360. Check out the [Display and Video (Actions)](/docs/connections/destinations/catalog/actions-display-video-360/) documentation if you want to set up a new instance of Google Display and Video 360. 
```
